src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -0,0 +1,100 @@
+    'Systems'
+];
+const SUGG_INSTANCE_TYPES_DATA = [
+    { instanceType: 't3.micro', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 't3.small', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 't2.small', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 2 },
+    { instanceType: 't2.micro', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 't2.large', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 4 },
+    { instanceType: 'm2.2xlarge', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 },
+    { instanceType: 'm3.2xsmall', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 6 },
+    { instanceType: 'm3.2xlarge', provider: 'AWS', description: 'Lorem ipsum dolor sit, amet consectetur adipisicing.', count: 1 }
+];
+
+export default function SuggestedInstanceTypesTable() {
+
+    const [page, setPage] = useState(PAGE);
+    const [perPage, setPerPage] = useState(PER_PAGE);
+
+    const onSetPage = (event, page) => setPage(page);
+    const onPerPageSelect = (event, perPage) => {
+        setPerPage(perPage);
+        setPage(PAGE);
+    };
+
+    return (
+        <>
+            <section className='pf-l-page__main-section pf-c-page__main-section'>
+                <Card>
+                    <CardBody>
+                        <PrimaryToolbar
+                            pagination={{
+                                itemCount: 50,
+                                page,
+                                perPage,
+                                onSetPage,
+                                onPerPageSelect
+                            }}
+                        />
+                        <TableComposable
+                            aria-label='suggested instance types table'
+                            variant={TableVariant.compact}
+
+                        >
+                            <Thead noWrap>
+                                <Tr>
+                                    {
+                                        SUGG_INSTANCE_TYPES_TABLE_COLUMNS.map(column => {
+                                            return <Th key={column}>{column}</Th>;
+                                        })
+                                    }
+                                </Tr>
+                            </Thead>
+                            <Tbody>
+                                {
+                                    SUGG_INSTANCE_TYPES_DATA.map(instance => {
+                                        return (
+                                            <Tr key={instance.instanceType}>
+                                                <Td>{instance.instanceType}</Td>
```

Comment:
@r14chandra can you please add `dataLabel` prop to Td tag and provide column name here for mobile view? you can read more about it here: https://www.patternfly.org/2022.11/components/table/

Reply:
@r14chandra also based on the mockup https://www.sketch.com/s/9070d288-aac5-4d00-aaac-ca899798ab00/a/dg0zx43 I feel we should reduce the width of first column, thoughts?

Reply:
> @r14chandra also based on the mockup https://www.sketch.com/s/9070d288-aac5-4d00-aaac-ca899798ab00/a/dg0zx43 I feel we should reduce the width of first column, thoughts?

I think that is fine. It is just a mockup and not a hard and fast rule to adapt exactly the same view.